packages/async-rewriter/src/async-writer-babel.spec.ts

```diff
@@ -929,25 +973,37 @@ describe('async-writer-babel', () => {
           db: signatures.Database
         });
       });
-      it('throws an error for db', () => {
+      it('throws an error for db', (done) => {
         try {
           writer.compile('fn(db)');
         } catch (e) {
           expect(e.name).to.be.equal('MongoshInvalidInputError');
+          done();
         }
       });
-      it('throws an error for db.coll', () => {
+      it('throws an error for db.coll', (done) => {
         try {
           writer.compile('fn(db.coll)');
```

Comment:
forgot to mention last time, i think is nice to add something like: `done(new Error('expected to throw'))` after this line. To avoid a timeout in case the test fails.

Reply:
Did you approve the PR otherwise?

Reply:
Cool, I think the done() fixes can be put off for later to save painful rebasing